recon/src/recon.rs

```diff
@@ -279,19 +280,12 @@ where
         self.store.value_for_key(&key).await
     }
 
-    /// Insert key into the key space. Includes an optional value.
-    /// Returns a boolean (true) indicating if the key was new.
-    pub async fn insert(&self, item: &ReconItem<'_, K>) -> Result<bool> {
-        self.store.insert(item).await
-    }
-
-    /// Insert many keys into the key space. Includes an optional value for each key.
-    /// Returns an array with a boolean for each key indicating if the key was new.
```

Comment:
we lost this per-key return value, do we not need that for anything?

Reply:
we never used it for anything before. the changes in #473 can still return it if we want it, but we really only care about the range query result after persisting things